docs/content/en/docs/getting-started/nutanix/nutanix-spec.md

```diff
@@ -278,7 +281,15 @@ Name of the project
  
 ### project.uuid (`name` or `UUID` required)
 UUID of the project
+
+### additionalCategories	(optional)
+Reference to a list of existing categories to be assigned to virtual machines. 
```

Comment:
Could you identify these as Nutanix categories and provide a link? Something like this: https://portal.nutanix.com/page/documents/details?targetId=Nutanix-Calm-Admin-Operations-Guide-v3_7_0:nuc-blueprints-tags-categories-overview-c.html

Reply:
Updated the documentation.